Assignment1/src/model/Hand.ts

```diff
@@ -1,18 +1,107 @@
-// Note: from tests I didn't understand what is "Hand" supposed to do?
-// Is it holding info about player's cards or handling moves for all players?
-// I assumed it's handling info, but idk. We will need to change Hand and Uno files anyway for them to work
-
-
 export interface Hand {
-    readonly playerName: string
-    readonly playerId: number
+    onEnd(arg: (e: any) => number): any;
+    score: any;
+    winner: any;
+    hasEnded: any;
+    sayUno(arg: number): any;
+    catchUnoFailure(arg0: { accuser: number; accused: number }): any;
+    playerCount: any;
+    player: any;
+    draw(): any;
+    canPlayAny(): any;
+    drawPile(): any;
+    canPlay: any;
+    discardPile(): any;
+    playerInTurn(): any;
+    playerHand: any;
+    play(number: number, color?: string): any;

```

Comment:
Other than wanting some actual types here I think it looks good, I will change the Uno.ts to match